explorer/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "author": {
+    "name": "Dusk Network B.V.",
+    "url": "https://dusk.network/",
+    "mail": "info@dusk.network"
+  },
+  "bugs": "https://github.com/dusk-network/rusk/issues",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "homepage": "https://github.com/dusk-network/rusk/tree/master/explorer",
+  "license": "MPL-2.0",
+  "name": "explorer",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dusk-network/rusk.git"
+  },
+  "scripts": {
+    "build": "vite build",
+    "checks": "npm run format && npm run lint && npm run typecheck && npm test",
+    "dev": "vite dev",
+    "dev:host": "vite dev --host",
+    "format": "prettier . --check",
+    "format:fix": "prettier . --write",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "preview": "vite preview",
+    "test": "vitest --run",
+    "test:coverage": "vitest --run --coverage.enabled",
+    "test:watch": "vitest",
+    "typecheck": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json --fail-on-warnings",
+    "typecheck:watch": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json --watch --fail-on-warnings"
+  },
+  "type": "module",
+  "version": "0.0.0",
+  "dependencies": {
+    "@mdi/js": "7.4.47"
+  },
+  "devDependencies": {
+    "@dusk-network/eslint-config": "3.1.0",
+    "@dusk-network/prettier-config": "1.1.0",
+    "@sveltejs/adapter-static": "3.0.1",
+    "@sveltejs/kit": "2.5.2",
+    "@testing-library/jest-dom": "6.4.2",
+    "@testing-library/svelte": "4.1.0",
+    "@types/node": "20.11.24",
+    "@vitejs/plugin-basic-ssl": "1.1.0",
```

Comment:
Maybe you don't want the hassle if you don't use the Crypto API, or need HTTPS for other reasons.